lib/order.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'text'
+
+class Order
+
+  attr_accessor :menu, :text
+
+  def initialize(text = Text_message.new)
+    @menu = @menu = {
+      "Sushi set" => 12,
+      "Prawn katsu curry" => 9,
+      "Kitsune udon" => 7,
+      "Onigiri" => 3,
+      "Matcha ice-cream" => 4
+    }
+    @total = 0
+    @order = []
+    @text = text
+  end
+
+  def print_menu
+    @menu.each { |key, value| puts " #{key}: £#{value}" }
+  end
+
+  def user_order(food)
+    if @menu.include?(food) == true
```

Comment:
When you have a conditional which looks like:
` if something == true` can be refactored to
`if something`

and 
`if something == false` can be refactored to
`if !something`